StackExchange.Redis/StackExchange.Redis.csproj

```diff
@@ -83,6 +84,7 @@
     <Compile Include="StackExchange\Redis\HashEntry.cs" />
     <Compile Include="StackExchange\Redis\IConnectionMultiplexer.cs" />
     <Compile Include="StackExchange\Redis\InternalErrorEventArgs.cs" />
+    <Compile Include="StackExchange\Redis\InternalRegexCompiledOption.cs" />

```

Comment:
This is actually a really cute solution to the compiled regex problem; I had a similar problem in a few other libs, and just settled for 5.2 vs 5.4 + dnx451 builds, but I'm very tempted to steal this, if you don't mind. Kudos.